### DISCUSSION TITLE
Lesson 12: ReentracyGuard.sol not available in openZeppelin packagee

### DISCUSSION THREAD

**Discussion by Ifra001** [Cyfrin/foundry-full-course-cu#2145] [Q&A] [answered]
Patrick got the ```ReentracyGuard.sol``` by following the path: openzeppelin-contracts -> contracts -> security -> ReentracyGuard

but when I installed the ```openzeppelin``` package the folder ```security``` wasn't there.

I don't know which pathways leads to ```ReentracyGuard.sol``` so plz help me out 

(I also looked into there Github repo and couldn't find it there)

**Comment by EngrPips** [comment, marked as answer]
Hello @DontMind-me, If you are on Mac, use `command + p`, and a search bar should appear. Type inside the search bar `ReentracyGuard` and `vscode` should point to where the file is if it exists in your working directory. If you are on Windows, use `cltr + p` instead and follow the same process. Let us know if that works for you

**Reply by Ifra001** [reply]
Thanks you bro! The ```ctrl + p``` command is a cool trick

**Reply by EngrPips** [reply]
Awesome, Keep crushing it.
